examples/user_interfaces/embedding_in_tk_canvas.py

```diff
@@ -1,63 +0,0 @@
-# -*- noplot -*-
```

Comment:
accidental delete in this push - it'll be back in the next one. However if the examples are ever built entirely from sphinx gallery, I don't believe these lines will be necessary anymore